### PR TITLE
PLATFORM-3117: simplified flow + some improvements

### PR DESCRIPTION
**Pull request by harnash**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +67,7 @@ This command will fetch all the variables from the defined sources and put them
 * **k8s-yaml** - will save configuration into Secret and ConfigMap YAMLs for use with kubectl
 * **envrc** - will save all configuration into shell compatible file for use in local development or testing
 
-When outputing `envrc` values with type `reference` will be omitted.
+When outpacing `envrc` values with type `reference` will be omitted.
```
Comment (themech) [inline README.md]: outpacing?

File: cmd/get.go [hunk 2]
```diff
@@ -23,21 +23,19 @@ import (
...
 
-// downloadCmd represents the download command
-var downloadCmd = &cobra.Command{
-	Use:   "download",
-	Short: "Downloads configuration and stores it locally",
-	Long:  `Fetches configuration for configured sources and stores it locally`,
+// getCmd represents the get command
+var getCmd = &cobra.Command{
+	Use:   "get",
+	Short: "Gets configuration and print out its contents",
```
Comment (themech) [inline cmd/get.go]: "prints out"

File: README.md [hunk 3]
```diff
@@ -62,7 +50,7 @@ Application:
 
 ## Available commands
 
-### download
+### get
 This command will fetch all the variables from the defined sources and put them in proper file(s).
```
Comment (themech) [inline README.md]: hmm, and the code below says it "Fetches configuration for configured sources and outputs it on stdout"

Comment (harnash) [inline README.md]: Fixed

File: cmd/get.go (hunk 2 above)
Comment (harnash) [inline cmd/get.go]: Fixed

File: README.md (hunk 1 above)
Comment (harnash) [inline README.md]: Fixed

**Pull request merged**